include/config/config_camera.h

```diff
@@ -44,6 +44,18 @@
  */
 #define PARALLEL_LAKITU_CAM
 
+/**
+ * Decides whether the camera sound that plays when rotating it is enabled, and which sound it is. 
+ * If you want the vanilla sound back, set it to SOUND_MENU_CAMERA_TURN
+ * Or you can disable the define to turn off the sound entirely.
+ */
+#define CAMERA_TURN_SOUND SOUND_MENU_MESSAGE_NEXT_PAGE
```

Comment:
While this sound is less annoying, I think it'd honestly throw me off since it's not really that subtle, so I'd just leave it default personally.

Reply:
idk, people might still want some audio feedback when turning the camera, without it being the horrible vanilla sound

Reply:
Ah heck I'm forgetting that zoom in is different from camera rotation too. Either way I stand by that all three of those sounds would need to be changed for it not to feel weird in-game. Additionally to be honest Idk if these need to be global defines either; four defines just for camera sounds is a bit bloaty IMO.

Reply:
with the number of people that change the camera sound I don't think it's too bloaty tbh. I've actually expected this define to be here in the past too lmao. 
and yeah zooming in and out is a pretty annoying sound as well. 
I'll give this a shot later this morning and see how this sound is since you mentioned it isn't that subtle. Both scut and hyena separately mentioned using it so it must be a decent improvement

Reply:
I'll do this. I insist on the default being the next page sound but I can live without doing that. But in that case I'm just turning the define off by default.

Reply:
I'm fine with it being default if it sounds good. I just wanna hear it first